taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -42,265 +41,14 @@ task-defaults:
         dummy-secrets:
             - content: "faketoken"
               path: .adjust_token
-    routes:
-        - notify.slack-channel.G016BC5FUHJ.on-failed
-    scopes:
-        - queue:route:notify.slack-channel.G016BC5FUHJ.on-failed
-            - notify:slack-channel:G016BC5FUHJ
-    extra:
-        notify:
-            # slackText: 'https://firefox-ci-tc.services.mozilla.com/tasks/${status.taskId} | ${task.metadata.name} | ${task.metadata.source}'
-            slackBlocks: |
-                [
-                  {
-                    "type": "header",
-                    "text": {
-                      "type": "plain_text",
-                      "text": "firefox-android :firefox: ${task.metadata.name} :x:\n "
-                    }
-                  },
-                  {
-                    "type": "divider"
-                  },
-                  {
-                     "type": "section",
-                     "text": {
-                         "type": "mrkdwn",
-                         "text": "*Task*: <https://firefox-ci-tc.services.mozilla.com/tasks/${status.taskId}|Taskcluster>"
-                    }
-                  },
-                  {
-                    "type": "section",
-                    "text": {
-                      "type": "mrkdwn",
-                      "text": "*Owner*: ${task.metadata.owner}"
-                    }
-                  },
-                  {
-                    "type": "section",
-                    "text": {
-                      "type": "mrkdwn",
-                      "text": "*Source*: <${task.payload.env.MOBILE_BASE_REPOSITORY}/commit/${task.payload.env.MOBILE_HEAD_REV}|Commit> :github:"
-                    }
-                  },
-                  {
-                    "type": "divider"
-                  },
-                  {
-                    "type": "context",
-                    "elements": [
-                        {
-                            "type": "mrkdwn",
-                            "text": ":testops-notify: created by <https://mozilla-hub.atlassian.net/wiki/spaces/MTE/overview|Mobile Test Engineering>"
-                        }
-                    ]
-                  }
-                ]
 
 tasks:
-    focus-arm-debug:
```

Comment:
Why are these tasks being deleted? Is that a separate issue?

Reply:
No no, it was just temporary because initially I didn't know how to stop them all from being triggered and was trying to only trigger one task for testing purposes.

They're all being added back.